db/controllers/articles.controllers.js

```diff
@@ -0,0 +1,49 @@
+const { selectArticleById, insertComment, selectCommentsByArticleId } = require('../models/articles.models');
+
+const getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+
+  selectArticleById(article_id)
+    .then(article => {
+      if (article) {
+        res.status(200).send({ article });
+      } else {
+        res.status(404).send({ msg: 'Article not found' });
```

Comment:
As per my other comments, this should be picked up in error handling middleware, not in the controller 👍